submarine-commons/commons-runtime/src/main/java/org/apache/submarine/common/BaseParameters.java

```diff
@@ -12,13 +31,11 @@
  * limitations under the License. See accompanying LICENSE file.
  */
 
-package org.apache.submarine.client.cli.param;
+package org.apache.submarine.common;
```

Comment:
Change to `package org.apache.submarine.commons;` ?

Reply:
@liuxunorg Thanks for reviewing. Maybe we'd better refactor the package name to `org.apache.submarine.commons.runtime`. Because it belongs to `commons-runtime` module. If okay, I will work for this ASAP.